templates/default/config/master.hcl.erb

```diff
@@ -30,3 +30,5 @@ telemetry = {
    prometheus_retention_time = "<%= node['consul']['metrics']['prometheus_retention_time'] %>",
    disable_hostname = true
 }
+log_file = "<%= node['consul']['logs_dir'] %>/consul"
```

Comment:
Why the `/consul` here?

Reply:
This value will be used as a prefix for the log file. Consul will append a timestamp to the log file above and the `.log` suffix. In the end it will be `/srv/hops/consul/logs/consul-1689587433046643271.log` The timestamp is used for the rotation. https://developer.hashicorp.com/consul/docs/agent/config/cli-flags#log-options